main.cpp

```diff
@@ -576,40 +650,22 @@ bool MonitorController::toggleCrossfeed(bool status)
 int main()
 {
     Example::main();
-    Cat Pusya;
-    SipProvider sipnet;
     SpaceShip Proton;
-    PowerUnit powerunit;
-    DAC dacLeft;
-    
-    std::cout << "\nSIP trunk status:" << std::endl;
-    sipnet.printStatus ();
-    
-    std::cout << "\n";
-    powerunit.printStatus();
     
-    std::cout << "\nDAC status: " << dacLeft.checkError(440) << std::endl;
+    for ( int i = 2; i < 5; ++i ) Proton.makeLoop(i, 6);
     
+    std::cout << Proton.totalLoops<< std::endl;
     
-    std::cout << "\nShip status: \n" ;
-    std::cout << ( Proton.dock()  ? "free fly" : "docked" ) << std::endl;
+    Knob volume;
     
-    std::cout << "\nCat attributes:" << "\n" 
-    << "colour: " << Pusya.colour << "\n"
-    << "age: " << Pusya.age << "\n"
-    << "gender: "<< ( Pusya.gender == 0 ? "Girl" : "Boy" ) << std::endl;
-    
-    Pusya.mew (3);
-    
-    // fill balance
-    sipnet.balance = 100;
-    std::cout << "\nSIP trunk status:" << std::endl;
-    sipnet.printStatus ();
+    for ( int i=0;i < 3; ++i )
+    {
+        std::cout << "Set volume: ";
+        std::cin >> volume.cvalue;
+        if ( volume.cvalue == volume.pvalue ) continue;

```

Comment:
Even if your `if/else/for/while/etc` expressions have only one statement, you should still put it on its own line.
- [ ] revise